scripts/declarations_diff.sh

```diff
@@ -1,5 +1,44 @@
 #!/usr/bin/env bash
 
+ : <<'BASH_DOC_MODULE'
+
+#  The `declarations_diff` script
+
+The `declarations_diff` script is a text-based script that attempts to find which declarations
+have been removed and which declarations have been added in the current PR with respect to `master`.
+
+In essence, it looks at the output of `git diff origin/master...HEAD`, it extracts the lines that
+contain one of the keywords
+
+`theorem` `lemma` `inductive` `structure` `def` `class` `instance` `alias`
+
+and tries to find exact matches between a removed line and an added line
+(e.g. when moving a declaration from one file to another or restructuring within the same file).
+
+Exact matches are removed.
+Among the remaining unmatched lines, the script tries to extract the declaration ids and tries to
```

Comment:
You could mention namespacing already up here, say like "the declaration ids as given by the diff --- in particular, not considering namespacing". What do you think?

Reply:
Good point: I added a sentence mentioning that the script ignores namespaces.